Superpowered/SuperpoweredAdvancedAudioPlayer.h

```diff
@@ -378,25 +378,32 @@ class AdvancedAudioPlayer {
 /// @param faster True: faster, false: slower.
 /// @param holdMs How long to maintain the pitch bend state. A value >= 1000 will hold until endContinuousPitchBend is called.
     JSWASM void pitchBend(float maxPercent, bool bendStretch, bool faster, unsigned int holdMs);
-        
+
 /// @brief Ends pitch bend.
     JSWASM void endContinuousPitchBend();
     
 /// @return Returns with the distance (in milliseconds) to the beatgrid while using pitch bend for correction.
     JSWASM double getBendOffsetMs();
+
+/// @return Returns with the current pitch bend percent. Will be 1 if there is no pitch bend happening.
+    JSWASM float getCurrentPitchBendPercent();
```

Comment:
This is new

Reply:
I suppose this will compile but if we try to use this method it'll blow up because it probably isn't in the version of the lib we are currently using?

Reply:
probably not a big deal

Reply:
Yea, I would imagine so. Can remove it to be safe if we want